# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../firefox/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Naur mozilla_orig/security/nss/lib/pki1/oiddata.h mozilla/security/nss/lib/pki1/oiddata.h
--- mozilla_orig/security/nss/lib/pki1/oiddata.h	2002-01-04 06:22:07.000000000 +0100
+++ mozilla/security/nss/lib/pki1/oiddata.h	2005-09-10 13:12:13.984991440 +0200
@@ -43,8 +43,8 @@
 #include "nsspki1t.h"
 #endif /* NSSPKI1T_H */
 
-extern const NSSOID nss_builtin_oids[];
-extern const PRUint32 nss_builtin_oid_count;
+//extern const NSSOID nss_builtin_oids[];
+//extern const PRUint32 nss_builtin_oid_count;
 
 /*extern const nssAttributeTypeAliasTable nss_attribute_type_aliases[];*/
 /*extern const PRUint32 nss_attribute_type_alias_count;*/
diff -Naur mozilla_orig/security/nss/lib/pki1/oidgen.perl mozilla/security/nss/lib/pki1/oidgen.perl
--- mozilla_orig/security/nss/lib/pki1/oidgen.perl	2001-11-08 01:15:25.000000000 +0100
+++ mozilla/security/nss/lib/pki1/oidgen.perl	2005-09-10 13:13:26.613950160 +0200
@@ -233,9 +233,6 @@
 #include "nsspki1t.h"
 #endif /* NSSPKI1T_H */
 
-extern const NSSOID nss_builtin_oids[];
-extern const PRUint32 nss_builtin_oid_count;
-
 /*extern const nssAttributeTypeAliasTable nss_attribute_type_aliases[];*/
 /*extern const PRUint32 nss_attribute_type_alias_count;*/
 
diff -Naur mozilla_orig/security/nss/lib/pki1/pki1.h mozilla/security/nss/lib/pki1/pki1.h
--- mozilla_orig/security/nss/lib/pki1/pki1.h	2001-07-19 22:40:42.000000000 +0200
+++ mozilla/security/nss/lib/pki1/pki1.h	2005-09-10 13:15:07.648590560 +0200
@@ -56,6 +56,8 @@
 PR_BEGIN_EXTERN_C
 
 /* fgmr 19990505 moved these here from oiddata.h */
+extern const NSSOID nss_builtin_oids[];
+extern const PRUint32 nss_builtin_oid_count;
 extern const nssAttributeTypeAliasTable nss_attribute_type_aliases[];
 extern const PRUint32 nss_attribute_type_alias_count;
 
